tests/integration/V1DeploymentsSpec.ts

```diff
@@ -0,0 +1,39 @@
+import {suite, test} from 'mocha-typescript';
+import {DynamoDBHelper} from '../helper/DynamoDBHelper';
+import Axios, {AxiosInstance} from 'axios';
+import * as assert from 'assert';
+import {Deployment} from '../../src/model/Deployment';
+
+interface IDeploymentFixture {
+  deployment: Deployment;
+}
+
+@suite
+export class V1DeploymentsSpec {
+
+  private readonly BASE_HOST = 'http://localhost:4000';
+  private readonly dynamoDBHelper: DynamoDBHelper;
+  private readonly httpClient: AxiosInstance;
+  private readonly ENDPOINT: string = '/v1/deployments';
+
+  constructor() {
+    this.dynamoDBHelper = new DynamoDBHelper();
+    this.httpClient = Axios.create({baseURL: this.BASE_HOST});
+  }
+
+  async before() {
+    await this.dynamoDBHelper.deleteSalesforceTable();
```

Comment:
You can rename `salesForce` :)

Reply:
will chnage it but yet it's not completed